build/theater.js

```diff
@@ -383,7 +485,7 @@
 
           newChar = speech.charAt(cursor);
 
-          if (--invincible < 0 && (prevChar !== newChar || self.current.experience < .3) && self.isMistaking()) {
+          if (--invincible < 0 && (prevChar !== newChar || self.current.experience <= .3) && self.isMistaking()) {
```

Comment:
I don't recall changing this.... I didn't mean to anyways. It's not relevant to this commit